templates/worker-deployment.yaml

```diff
@@ -31,10 +31,16 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.worker.template.spec.containers.image.name }}:{{ .Values.worker.template.spec.containers.tag }}"
+          image: "{{ .Values.worker.repository }}:{{ .Values.worker.repository_tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
-            {{.Values.worker.template.spec.containers.env}}    
+          {{- range .Values.worker.template.spec.containers }}
```

Comment:
i don't think this makes sense, you have a nested loop here, don't think the env vars should be dependent on `.Values.worker.template.spec.containers`